modules/ossm-rn-fixed-issues.adoc

```diff
@@ -23,6 +23,10 @@ The following issues been resolved in the current release:
 +
 Using {SMProductName} Operator 2.2 or 2.3, the SMMR controller no longer removes the namespaces from `SMMR.status.configuredMembers`. Instead, the controller adds the namespaces to `SMMR.status.pendingMembers` to indicate that they are not up-to-date. During reconciliation, as each namespace synchronizes with the SMCP, the namespace is automatically removed from `SMMR.status.pendingMembers`.
 
+* https://issues.redhat.com/browse/OSSM-1962[OSSM-1962] Use `EndpointSlices` in federation controller.
++
+The federation controller now uses `EndpointSlices`, which improves scalability and performance in large deployments. The PILOT_USE_ENDPOINT_SLICE flag is enabled by default. Disabling the flag prevents use of federation deployments.
```

Comment:
I believe the last sentence is wrong. Setting the flag to false simply causes the controller to use Endpoints instead of EndpointSlices. @jewertow please confirm.

Reply:
No, this is correct. I didn't implement fallback to Endpoints. All versions of OCP on which OSSM 2.3 will be supported, support EndpointSlices, so I didn't see any reason to still rely on Endpoints.

Reply:
It sounds like no doc change is needed.